frontend/src/streamlist/__tests__/StreamsTable.test.jsx

```diff
@@ -108,95 +107,173 @@ describe("<StreamsTable/> component", () => {
+          it(`should have title "${expected[7].title}"`, () => {
+            expect(result[7].title).toBe(expected[7].title);
+          });
+          it(`should have field "${expected[7].field}"`, () => {
+            expect(result[7].field).toBe(expected[7].field);
+          });
+          it(`should have filtering "${expected[7].filtering}"`, () => {
+            expect(result[7].filtering).toBe(expected[7].filtering);
+          });
+          it(`should have sorting "${expected[7].sorting}"`, () => {
+            expect(result[7].sorting).toBe(expected[7].sorting);
+          });
+          it(`should have align "${expected[7].align}"`, () => {
+            expect(result[7].align).toBe(expected[7].align);
+          });
+          it(`should have export "${expected[7].export}"`, () => {
+            expect(result[7].export).toBe(expected[7].export);
+          });
+          it(`should have a render() function that returns a <StreamDetailsButton/> component`, () => {
+            const dummyData = {
+              streamInfo: dummyStreams[0]
+            };
+            const renderResult = result[7].render(dummyData);
+            expect(renderResult).toMatchObject(expected[7].render(dummyData));
+          });
         });
       });
-      describe("index [7] info", () => {
-        it(`should have title "${expected[7].title}"`, () => {
-          expect(result[7].title).toBe(expected[7].title);
-        });
-        it(`should have field "${expected[7].field}"`, () => {
-          expect(result[7].field).toBe(expected[7].field);
-        });
-        it(`should have filtering "${expected[7].filtering}"`, () => {
-          expect(result[7].filtering).toBe(expected[7].filtering);
-        });
-        it(`should have sorting "${expected[7].sorting}"`, () => {
-          expect(result[7].sorting).toBe(expected[7].sorting);
+    });
+    describe("if isSimple prop is true", () => {
+      wrapper = Enzyme.shallow(
+        <StreamsTable streams={dummyStreams} isSimple />
+      );
+      const result = wrapper.instance().getColumnInfo();
+
+      describe("Returns an array of objects", () => {
+        describe("index [0] info", () => {
+          it(`should have title "${expectedSimple[0].title}"`, () => {
+            expect(result[0].title).toBe(expectedSimple[0].title);
+          });
+          it(`should have field "${expectedSimple[0].field}"`, () => {
+            expect(result[0].field).toBe(expectedSimple[0].field);
+          });
         });
-        it(`should have align "${expected[7].align}"`, () => {
-          expect(result[7].align).toBe(expected[7].align);
+        describe("index [1] info", () => {
+          it(`should have title "${expectedSimple[1].title}"`, () => {
+            expect(result[1].title).toBe(expectedSimple[1].title);
+          });
+          it(`should have field "${expectedSimple[1].field}"`, () => {
+            expect(result[1].field).toBe(expectedSimple[1].field);
+          });
```

Comment:
One test per property per index of columnInfo is overkill imo. Just make one test that takes the return value of `getColumnInfo` and does all these assertions. It'll make this test file waaaay more concise.